packages/shopify-react/README.md

```diff
@@ -0,0 +1,122 @@
+<div align="center">
+  <p>
+    <img alt="Gadget logo" src="https://raw.githubusercontent.com/gadget-inc/js-clients/main/docs/assets/gadget-logo.png" />
+  </p>
+  <p>
+    <a href="">
+      <img alt="GitHub CI status" src="https://badgen.net/github/checks/gadget-inc/js-clients/main/Test?label=CI" />
+    </a>
+    <a href="https://www.npmjs.com/package/@gadgetinc/shopify-react">
+      <img alt="npm version" src="https://badgen.net/npm/dw/@gadgetinc/shopify-react?color=4148f2" />
+    </a>
+  </p>
+  <p>
+    <strong>
+      React hooks for use with application APIs built using the <a href="https://gadget.dev">Gadget</a> platform.
+    </strong>
+  </p>
+</div>
+
+## Features
+
+This library wraps [@shopify/app-bridge](https://www.npmjs.com/package/@shopify/app-bridge) and [@shopify/app-bridge-react](https://www.npmjs.com/package/@shopify/app-bridge-react) to allow you to easily communicate both with the Shopify admin and your Gadget backend.
+
+## Installation
+
+This is a companion package to the JavaScript client package generated for your Gadget app, so you must install the JS client for your app, and then install this package.
+
+To install the JS client for your app, you must set up the Gadget NPM registry, and then install the client:
+
+```bash
+npm config set @gadget-client:registry https://registry.gadget.dev/npm
+yarn add @gadget-client/my-app-slug
+# or
+npm install @gadget-client/product-tagger-8000
+```
+
+Full installation instructions can be found in the Gadget docs at `https://docs.gadget.dev/api/<my-app-slug>/installing`.
+
+Once you have your JS client installed, you can install the React hooks library with yarn or npm:
+
+```
+yarn add @gadgetinc/shopify-react @gadgetinc/react @shopify/app-bridge-react react
+# or
+npm install --save @gadgetinc/shopify-react @gadgetinc/react @shopify/app-bridge-react react
+```
+
+## Example usage
```

Comment:
Can we add some explicit notes about not following the instructions in the shopify docs and in the `@gadget/react` docs